editor/src/components/canvas/canvas-strategies/strategies/grid-helpers.ts

```diff
@@ -70,6 +72,10 @@ export function gridPositionToValue(
   return p.numericalPosition
 }
 
+export function isAutoGridPin(v: GridPositionOrSpan): boolean {
+  return isCSSKeyword(v) && v.value === 'auto'
+}
+
 export function setGridPropsCommands(
```

Comment:
these command generator functions are usually called get * commands or create * commands. And since this is not setting _all_ grid props, just the placement are props, I would rename it to something like `getCommandsForGritItemPlacement`

Reply:
renamed in [`b75261c` (#6633)](https://github.com/concrete-utopia/utopia/pull/6633/commits/b75261c5bf83c93509dd92833d87d9ad2d61ee0c)